src/Categories/Category/Construction/Quivers.agda

```diff
@@ -0,0 +1,191 @@
+{-# OPTIONS --without-K --safe #-}
+
+module Categories.Category.Construction.Quivers where
+
+-- The "Underlying Graph" <-> "Free Category on a Graph" Adjunction.
+--   [It's actually a Multidigraph, or Quiver.  Use the latter.]
+-- Lots of surprises here, of various level of surprisingness
+-- 1. The Free Category rises universe levels of arrows (and equivalences); the Underlying Graph does not
+--   (due to the Transitive Closure "Star" construction)
+-- 2. As a consequence of (1), the eventual Adjunction will be for all Levels being the same
+-- 3. If you want a Functor Categories Graph, then the object/hom levels are straightforward,
+--   but if you use "Categories" with NaturalIsomorphism, then the 2-point Groupoid is contractible
+--   (thus equivalent to the 1-point Category), and yet those two Graphs shouldn't be considered to
+--   be the same!  So _≡F_ must be used so that we have no wiggle room when mapping Graphs around.
+--   Interestingly, this only shows up when trying to prove that the Underlying Functor respects
+--   _≈_, which is not traditionally a requirement of a Functor! This requirement is usually left
+--   implicit.  If we were to allow Functor to not respect _≈_, things would be fine. But of course
+--   we couldn't do any reasonable equational reasoning anymore.  Of course, if _≈_ were the global
+--   _≡_, this isn't an issue, but that is definitely not wanted here.
+-- 4. A lot of weird little lemmas about _≡_ and subst/transport are needed, because we're in a mixed
```

Comment:
Not sure this comment applies any longer after this reorganization.

Reply:
In the construction of the category, no. But in the constructor of the functor, it does, so I've adjusted the comments there.